### PR TITLE
Sign Up Page Update - Pull Request

### PR DESCRIPTION
**Pull request by Aboudik1452**
### The changes for this PR are listed below.
- Removed verify e-mail box & verify password box;
- Added the option to select the account type;
- Centered the create account button.

### CODE REVIEW

File: src/Components/Pages/SignUpPage.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
-import { Button, Card, Container, Form } from 'react-bootstrap';
+import { Card, Container, Button, Dropdown, DropdownButton, Form } from 'react-bootstrap';
```
Comment (nolanbastien) [inline src/Components/Pages/SignUpPage.js:1]: Make sure you don't have unused import statements.

**Review by nolanbastien — CHANGES_REQUESTED** [review]
Fix imports.

File: src/Components/Pages/SignUpPage.js (hunk 1 above)
Comment (Aboudik1452) [reply src/Components/Pages/SignUpPage.js]: Removed unused libraries.

**Review by ThaCovenant — APPROVED** [review]
Joe you're numbah 1

**Pull request merged**